src/components/Graph/Graph.tsx

```diff
@@ -90,6 +90,16 @@ const Graph: FC<GraphProps> = memo(({ pathway, interactive = true, expandCurrent
     });
   }
 
+  // Find node that is farthest to the right
+  const maxWidth = useMemo(() => {
+    return nodeCoordinates !== undefined
+      ? Object.values(nodeCoordinates)
+          // Add 250 to account for expanded node width
+          .map(x => x.x + parentWidth / 2 + 250)
```

Comment:
Is 250 already defined as or based on a constant somewhere? Or is this just based on experimentation? The comment is good but if we can link it to something more concrete that would be good.

Reply:
Mostly based on experimentation. The min width for an expanded node is 400 so 250 was chosen to cover arrows drawn slightly past the halfway point. I'll try to find something more concrete to link it to.

Reply:
Ok I changed this to use the width of the node instead of the random 250 number.